static/app/views/admin/options.tsx

```diff
@@ -261,7 +261,7 @@ const definitions: Field[] = [
       ['true', 'Please keep my usage information anonymous'],
     ],
     help: tct(
-      'If enabled, any stats reported to sentry.io will exclude identifying information (such as your administrative email address). By anonymizing your installation the Sentry team will be unable to contact you about security updates. For more information on what data is sent to Sentry, see the [link:documentation].',
+      'If enabled, any stats reported to sentry.io will exclude identifying information (such as your administrative email address). By anonymizing your installation the Sentry team will be unable to contact you about security updates. For more information on what data is sent to Sentry, see the [link:documentation]. Note: This is separate from error-reporting for the self-hosted installer.',
```

Comment:
I think it would be good to explain how this is different than the installer reporting, not just saying it is different. (Maybe emphasize the beacon is for the running instance, while the installer error reporting is just for the installer).

Reply:
good point